### ISSUE TITLE
Missing events?

### ISSUE THREAD

**Issue by dgarciabriseno** [Helioviewer-Project/api#180]
```
====[DATE]====================

2022/07/18 05:54:00

====[SOURCE]==================

api.helioviewer.org/v2/takeScreenshot/?date=2022-04-26T23:59:59Z&imageScale=8&scale=false&layers=[STEREO_A,SECCHI,EUVI,304,1,100]&events=[AR,HMI_HARP;SPoCA,1],[CH,all,1]&eventsLabels=false&x0=0&y0=0&width=300&height=300&display=true&watermark=true

====[MESSAGE]=================

unable to open image `/var/www/api.helioviewer.org/docroot/cache/events/2022/04/26/ivo%3A%2F%2Fhelio-informatics.org%2FCH_SPoCA_20220428_000131_20220427T031553_9.png': No such file or directory @ error/blob.c/OpenBlob/2881

====[POST]=================

'action' => takeScreenshot
'date' => 2022-04-26T23:59:59Z
'imageScale' => 8
'scale' => false
'layers' => [STEREO_A,SECCHI,EUVI,304,1,100]
'events' => [AR,HMI_HARP;SPoCA,1],[CH,all,1]
'x0' => 0
'y0' => 0
'width' => 300
'height' => 300
'display' => true
'watermark' => true
```

We have event data for 2022-04-26
but we do not have this png `ivo%3A%2F%2Fhelio-informatics.org%2FCH_SPoCA_20220428_000131_20220427T031553_9.png`

**Comment by wafels** [comment]
How do we update features and events? It should be something similar to how we go back and fill in missing images.

**Comment by dgarciabriseno** [comment]
I am pretty sure it's the `importEvents` function I'm working on. Let me try running it locally for 2022-04-26 and see if I get that png

**Comment by dgarciabriseno** [comment]
Yep, requesting events again gets this png. I'm not sure what happened, but it looks like 3 events are missing for 2022-04-26 on helioviewer.org.

**Comment by dgarciabriseno** [comment]
Once #153 is done we'll be able to backfill targeted dates.

**Comment by dgarciabriseno** [comment]
Finally coming back around to this. I discovered that while the backfill will update the database, it doesn't actually add new pngs because of unix permissions problems. I should know better by now, there are always permissions problems.

Anyway, I fixed the above specific issue, I'm running a process now to find all of these event images that we may have in the database but not have the images for. This way I can run a targeted backfill to repair any similar problems.

**Comment by dgarciabriseno** [comment]
The process I was running did not return the results I wanted from it... So this is still a problem. It prevents movies from being created over specific time ranges. I just repaired 2014-04-07, but I need a way of determining which files are missing. Which is what the process I was running was supposed to do.

**Comment by dgarciabriseno** [comment]
I don't remember what script I was running before, but looking through logs, I don't see this error happening anymore so whatever I did must have worked.
